.eslintrc

```diff
@@ -38,7 +38,8 @@
                 "args": "none"
             }
         ],
-        "comma-dangle": 0,
+        "no-console": 1,                      // 1=warn
+        "comma-dangle": 0,                    // 0=off
```

Comment:
I guess when I refer to their docs, I go to the rule: http://eslint.org/docs/rules/comma-dangle and their rules mention nothing about the options for settings it, just the specifics.